lib/eigensolve_quda.cpp

```diff
@@ -191,6 +191,42 @@ namespace quda
     delete tmp2;
   }
 
+  double EigenSolver::estimateChebyOpMax(const DiracMatrix &mat, ColorSpinorField &out, ColorSpinorField &in)
+  {
+
+    if (in.Location() == QUDA_CPU_FIELD_LOCATION) {
+      in.Source(QUDA_RANDOM_SOURCE);
+    } else {
+      RNG *rng = new RNG(in, 1234);
```

Comment:
While it sort of doesn't matter, I think we should use a different "magic number" whenever we create an RNG. I know we use 1234 elsewhere.

Food for thought @maddyscientist @cpviolator , I'm definitely not going to hold up the PR on this.